skiko/src/awtMain/cpp/windows/directXRedrawer.cc

```diff
@@ -354,7 +354,7 @@ extern "C"
     }
```

Comment:
Are we allow such single word commit message as "rename"?

Reply:
It will be squashed on merge anyway. And this commit is actually rename